scripts/kind.sh

```diff
@@ -19,7 +19,7 @@ set -e
 
 UPLOAD_IMAGES=
 TAG=latest
-KUBEVER=1.21.1
+KUBEVER=1.23.0
```

Comment:
I'm developing in my WSL2 environment, and for some weird reasons 1.21.1 would lead to failure of kind cluster creation.
See [this](https://github.com/kubernetes-sigs/kind/issues/2323).

Reply:
Okay, lets leave this then.  But this does impact all of the e2e runs that we do in the CI.  So, lets call this out in the PR description.